### PR TITLE
[v2/bug] Handle long responses utilising the Imagine module

### PR DESCRIPTION
**Pull request by daniwasonline**
target: v2.0.1

# Description
This PR adds a simple switch. When the response is over 2,000 characters and the Imagine module is being called, the final response should automatically place the final response into a file as well.

### CODE REVIEW

**Comment by daniwasonline** [comment]
Again, this is bound for 2.1; this should stay open until we get 2.0 merged into main.

File: src/util/models/interactions.js [hunk 1]
```diff
@@ -391,13 +391,19 @@ export class InteractionMessageEvent {
 
 		return responseMsg
 			.edit({
-				content: final,
+				content: final?.trim()?.length >= 2000 ? "" : final,
 				files: [
+					final?.trim()?.length >= 2000
+						? {
+								attachment: Buffer.from(final, "utf-8"),
```
Comment (artifishvr) [inline src/util/models/interactions.js:398]: do we have to update this now cause of #69

Comment (daniwasonline) [reply src/util/models/interactions.js:398]: fuck.

Comment (daniwasonline) [reply src/util/models/interactions.js:398]: wait no, there's no support for long response links in passive activation yet (which is what this is about)

Comment (artifishvr) [reply src/util/models/interactions.js:398]: oh ok

**Review by artifishvr — APPROVED** [review]

**Pull request merged**